README.md

```diff
@@ -66,20 +66,13 @@ enum ScreenIdleState {
     "unlocked"
 };
 
-[
-  SecureContext,
-  Exposed=(Window,DedicatedWorker),
-] interface IdleState {
-  readonly attribute UserIdleState user;
-  readonly attribute ScreenIdleState screen;
-};
-
 [
   SecureContext,
   Exposed=(Window,DedicatedWorker)
 ] interface IdleDetector : EventTarget {
   constructor(optional IdleOptions options = {});
-  readonly attribute IdleState state;
+  readonly attribute UserIdleState? userState;
```

Comment:
Why nullable?

Reply:
Until `start()` is called these attributes don't have a defined value. In the Chromium implementation the `state` attribute could have be null and this was allowed due to a bug in Blink's WebIDL bindings. I've formalized that here. Note that #10 suggests a way of removing this state by construction.

Reply:
thx!